app/_hub/kong-inc/jwt/index.md

```diff
@@ -421,24 +439,26 @@ This private key must be kept secret. To generate a public key corresponding to
 $ openssl rsa -in private.pem -outform PEM -pubout -out public.pem
 ```
 
-If you run the commands above, the public key will be written in `public.pem`, while the private key will be written in `private.pem`.
+If you run the commands above, the public key is written in `public.pem`, whereas the
+private key is written in `private.pem`.
 
 ### Using the JWT plugin with Auth0
 
 [Auth0](https://auth0.com/) is a popular solution for Authorization, and relies
 heavily on JWTs. Auth0 relies on RS256, does not base64 encode, and publicly
 hosts the public key certificate used to sign tokens. Account name is referred
-to "COMPANYNAME" for the sake of the guide.
+to as `{COMPANYNAME}` for the sake of the following examples.
```

Comment:
A question rather than a suggestion: We use `<variable>` and `{variable}` all over the docs to indicate variables that can be changed. We should settle on one consistent style here. For autogenerated examples, I used `< >`, however now I look at the Admin API docs (for example) and see curly brackets all over the place. Any reason to use one over the other?

Reply:
> A question rather than a suggestion: We use `<variable>` and `{variable}` all over the docs to indicate variables that can be changed. We should settle on one consistent style here. For autogenerated examples, I used `< >`, however now I look at the Admin API docs (for example) and see curly brackets all over the place. Any reason to use one over the other?

@lena-larionova  Yes, this convention is inconsistent in our docs. Note that our paths use curly brackets for service, route consumer:

curl -i -X POST http://localhost:8001/consumers/**{consumer}**/jwt \
    -F "algorithm=RS256" \
    -F "rsa_public_key=@./pubkey.pem" \
    -F "key=https://**{COMPAYNAME}**.auth0.com/" # the `iss` field

and I just noticed a typo. I'm fine with using a consistent approach for placeholder data throughout the docs. @GayleNeumann something to discuss and put into a style guide.

Another thing I'd like to mention--now there are 3 versions of the plugin source files, and the edits made in this file should probably be backported to the other versions. I will hold off for now unless otherwise instructed. If we had a properly versioned repo, we could accomplish this easily with a few cherry-pick, preferably automated with a merge script. Not having a properly versioned docs repo causes inconsistencies between versions.